Lib/asyncio/locks.py

```diff
@@ -95,6 +95,9 @@ async def acquire(self):
         This method blocks until the lock is unlocked, then sets it to
         locked and returns True.
         """
+        # implement fair scheduling, where thread always waits
```

Comment:
Capitalize sentence.
```suggestion
        # Implement fair scheduling, where thread always waits
```
(And thanks for adding the comments!)